src/components/settings/SettingsPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import './settingsPage.css';
+
+class SettingsPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+  
+  render() {
+    return (
+      <div>
+        <h1 className="settings-title">Settings</h1>
+        <div className="container">
+          <p>Add New User</p>
+          <img id="logo" src="https://pbs.twimg.com/profile_images/1093256137233715200/BVZAAvRi_400x400.jpg">
```

Comment:
This is not the correct logo. I believe the project has the logo image, it should be under the folder `images`, maybe check that out :)